docs/_appendix/reverting.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Reverting Changes
+order: 3
+---
+Testing is imperfect and from time to time changes are merged into master which
+break things for users of web-platform-tests. Such breakage can include:
+
+  * Failures in Travis or Taskcluster runs for this repository, either on the
+    master branch or on pull requests following the breaking change.
+
+  * Breakage in browser engine repositories which import and run
+    web-platform-tests, such as Chromium, Edge, Gecko, Servo and WebKit.
+
+  * Breakage in results collections systems for results dashboards, such as
+    [wpt.fyi](https://wpt.fyi).
+
+  * Breakage in supplemental tooling used by working groups, such as the
+    [CSS build system][].
```

Comment:
url?

Reply:
oh sorry, I don't know markdown syntax it seems.

Reply:
This isn't even Markdown I'm afraid, it's Jekyll. Quite easy to get things wrong :)